auacm/app/modules/blog_manager/blog_test.py

```diff
@@ -35,59 +33,57 @@ def testCreate(self):
                 self.assertEqual(post[key], post_response[key])
 
         # Delete the post from the test database
-        session.flush()
-        post = session.query(BlogPost).filter(
-            BlogPost.id == post_response['id']).first()
-        session.delete(post)
-        session.commit()
+        DATABASE_SESSION.flush()
+        post = DATABASE_SESSION.query(BlogPost).filter(
+            BlogPost.post_id == post_response['id']).first()
+        DATABASE_SESSION.delete(post)
+        DATABASE_SESSION.commit()
 
-    def testGetAll(self):
```

Comment:
Judging from the `unittest` docs, it looks like the standard is more or less a mix: use `snake_case` except in standard methods like `setUp()`. So this is fine